test/e2e/scale.go

```diff
@@ -127,7 +128,7 @@ func ScaleToWithin(t *testing.T, scale int, duration time.Duration, latencies La
 				return v1a1test.IsServiceReady(s)
 			}, "ServiceUpdatedWithURL")
 			if err != nil {
-				t.Errorf("WaitForServiceState(w/ Domain) = %v", err)
+				// t.Errorf("WaitForServiceState(w/ Domain) = %v", err)
```

Comment:
why comment?

Reply:
Thanks for the review! i'm still in the process of debugging this with prow, so this as well as the log statements with starts are for debug purposes. I will clean this up once I'm done.